config/frontend.ssl.properties

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2016 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+# rest server
+rest.server.blob.storage.service.factory=com.github.ambry.frontend.AmbryBlobStorageServiceFactory
+rest.server.router.factory=com.github.ambry.router.NonBlockingRouterFactory
```

Comment:
yes, I have added both coordinatorbacked router and NBRouter. But then, we need to remember that leaving it to default will not enable SSL, as default in "InMemoryRouter" and hence not network interactions will be triggered.